cmd/playground/window_state.go

```diff
@@ -0,0 +1,75 @@
+package playground
+
+import (
+	"time"
+
+	"github.com/odpf/optimus/job"
+)
+
+type state struct {
+	windowV1      job.WindowV1
+	windowV2      job.WindowV2
+	scheduledTime time.Time
+}
+
+// increment the component of the time which the cursor is pointed at
+func (s *state) IncrementTime(increaseBy string) {
+	switch increaseBy {
+	case pointToMinute.getStringValue():
+		s.scheduledTime = s.scheduledTime.Add(time.Minute)
+	case pointToHour.getStringValue():
+		s.scheduledTime = s.scheduledTime.Add(time.Hour)
+	case PointToDay.getStringValue():
+		s.scheduledTime = s.scheduledTime.AddDate(0, 0, 1)
+	case pointToMonth.getStringValue():
+		s.scheduledTime = s.scheduledTime.AddDate(0, 1, 0)
+	case pointToYear.getStringValue():
+		s.scheduledTime = s.scheduledTime.AddDate(1, 0, 0)
+	}
+}
+
+// decrement the component of the time which the cursor is pointed at
+func (s *state) DecrementTime(decreaseBy string) {
+	switch decreaseBy {
+	case pointToMinute.getStringValue():
+		s.scheduledTime = s.scheduledTime.Add(-1 * time.Minute)
+	case pointToHour.getStringValue():
+		s.scheduledTime = s.scheduledTime.Add(-1 * time.Hour)
+	case PointToDay.getStringValue():
+		s.scheduledTime = s.scheduledTime.AddDate(0, 0, -1)
+	case pointToMonth.getStringValue():
+		s.scheduledTime = s.scheduledTime.AddDate(0, -1, 0)
+	case pointToYear.getStringValue():
+		s.scheduledTime = s.scheduledTime.AddDate(-1, 0, 0)
+	}
+}
+
+// change the value of truncate
+func (s *state) IncrementTruncate() {
```

Comment:
If no one uses this outside, just make it as unexported (small `I`) and please update the others as well